explore/src/main/scala/explore/config/SpectroscopyModesTable.scala

```diff
@@ -387,7 +387,7 @@ object SpectroscopyModesTable extends ItcColumn {
         )
       )((props, _, itcResults, _, _, _, _, _) => { case (wv, sn, rows, range) =>
         implicit val ctx = props.ctx
-        range.value.map(r => updateITCOnScroll(wv, sn, r, rows, itcResults.withEffect)).orEmpty
+        range.value.map(r => updateITCOnScroll(wv, sn, r, rows, itcResults)).orEmpty
```

Comment:
???

Reply:
Ooops, sorry. I was trying things out and this slipped through.

This is unrelated, but it seems the `withEffect` is not required.